package.json

```diff
@@ -31,7 +31,7 @@
     "homepage": "https://github.com/pokt-network/testnet-faucet#readme",
     "dependencies": {
         "@koa/router": "^8.0.5",
-        "@pokt-network/pocket-js": "0.0.1-rc",
+        "pocket-js": "pokt-network/pocket-js#staging",
```

Comment:
While this was a good solution 5 minutes ago, I think we should wait for 0.1.1-RC on NPM since this is a tag that could bite us in the ass if we forget about it next time.